src/utils/utils.ts

```diff
@@ -28,3 +28,26 @@ export const imageURL = (
 ) => {
   return `https://iiif.nypl.org/iiif/2/${imageId}/${region}/${size}/${rotation}/default.jpg`;
 };
+
+export const getNumItems = async (uuid: string) => {
```

Comment:
Add jsdoc style comment to this function, similar to the `imageURL` comment block above.